src/main/io/serial_cli.c

```diff
@@ -298,253 +298,364 @@ const clicmd_t cmdTable[] = {
 };
 #define CMD_COUNT (sizeof(cmdTable) / sizeof(clicmd_t))
 
+static const char * const lookupTableOffOn[] = {
+    "OFF", "ON"
+};
+
+static const char * const lookupTableUnit[] = {
+    "IMPERIAL", "METRIC"
+};
+
+static const char * const lookupTableAlignment[] = {
+    "DEFAULT",
+    "CW0",
+    "CW90",
+    "CW180",
+    "CW270",
+    "CW0FLIP",
+    "CW90FLIP",
+    "CW180FLIP",
+    "CW270FLIP"
+};
+
+#ifdef GPS
+static const char * const lookupTableGPSProvider[] = {
+    "NMEA", "UBLOX"
+};
+
+static const char * const lookupTableGPSSBASMode[] = {
+    "AUTO", "EGNOS", "WAAS", "MSAS", "GAGAN"
+};
+#endif
+
+static const char * const lookupTableCurrentSensor[] = {
+    "NONE", "ADC", "VIRTUAL"
+};
+
+static const char * const lookupTableGimbalMode[] = {
+    "NORMAL", "MIXTILT"
+};
+
+static const char * const lookupTablePidController[] = {
+    "MULTIWII", "REWRITE", "LUX", "MULTIWII23", "HARAKIRI"
+};
+
+static const char * const lookupTableSerialRX[] = {
+    "SPEK1024",
+    "SPEK2048",
+    "SBUS",
+    "SUMD",
+    "SUMH",
+    "XB-B",
+    "XB-B-RJ01"
+};
+
+
+typedef struct lookupTableEntry_s {
+    const char * const *values;
+    const uint8_t valueCount;
```

Comment:
You can NULL-terminate value lists and avoid this structure (memory requirements are the same due to padding, code will be probably few bytes shorter).

It is also possible to store pointer to lookup table directly - the min/max union is large enough, so there is no memory penalty.

The code may be a even a bit easier to read .. but that's mainly  mater of taste ..